.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        version: [10.x, 12.x]
+        version: [10.x, 12.x, 14.x, 16.x]
```

Comment:
Thanks for adding support for 14 too. I didn't realize it hadn't been added. You can also get rid of 10.x since we no longer support it.